vendor/github.com/prometheus/alertmanager/silence/silence.go

```diff
@@ -204,8 +204,8 @@ type Silences struct {
 
 // Limits contains the limits for silences.
 type Limits struct {
-	// MaxSilences limits the maximum number of silences, including expired
-	// silences.
+	// MaxSilences limits the maximum number active and pending silences.
```

Comment:
This is a regression I'm afraid. Please see https://github.com/prometheus/alertmanager/commit/dbe6312f098f6698c3b82ed736c0c165a0e19f6a.